test/backend/test_node.py

```diff
@@ -1446,9 +1446,11 @@ def _test_pooling(self,
                     ceil_mode=None,
                     count_include_pad=None,
                     pooling_type="MAX",
-                    input_dtype=np.float32):
+                    input_dtype=np.float32,
+                    p=2):
 
-    op = "MaxPool" if pooling_type.upper().startswith("MAX") else "AveragePool"
+    op = "MaxPool" if pooling_type.upper().startswith("MAX") else \
+         "AveragePool" if pooling_type.upper()=="AVG" else "LpPool"
     node_def_kwargs = {
         "op_type": op,
         "inputs": ["X"],
```

Comment:
We probably need to add p to node_def_kwargs too to test lp_pool?

Reply:
Thanks for catching this. Fixed in the new commit.